fixed respos Requests and viewmodels and Made 3 adapters and viewholders

dovahcin opened the pull request.

dovahcin committed:
fixed respos Requests and viewmodels and Made 3 adapters and viewholders

rezaiyan commented on app/src/main/res/layout/item_horizontalmoviefeature.xml:
```diff
@@ -10,14 +12,15 @@
             android:layout_weight="1"
             android:adjustViewBounds="true"
-            app:shapeAppearanceOverlay="@style/cornerdImage"
+            android:src="@{picture.poster_path}"
```
پارمتر picture تعریف نشده

dovahcin replied:
آره هنوز نرسیده بودم اونجا

rezaiyan replied:
👍
وقتی کدی پوش میشه بهتره قابل اجرا باشه و کامپایلر خطا نده

dovahcin committed:
Fixed server responses
and loaded data for views :
details page
similar movies
recommendation movies
genres
and completed the layout Ui
except Recommendation movies Overview

rezaiyan commented on app/src/main/java/com/movie/android/data/ApiServices.kt:
```diff
@@ -16,21 +15,19 @@ interface ApiServices {
     ): Details
 
-    @GET("/movie/{movie_id}/similar")
+    @GET("/3/movie/{movie_id}/similar")
```
😅

dovahcin replied:
ا🤣🤣🤣🤣🤦‍♂️🤦‍♂️ آقا چیکار کنم دیباگ زیادی مغزو stackoverflow میکنه

dovahcin commented on app/src/main/res/layout/fragment_details.xml:
```diff
@@ -3,146 +3,201 @@
+        tools:context=".view.fragment.DetailsFragment">
+
+
+        <ImageView
```
واقعیتش دیگه انرژیم نموند این imageview رو هاله سیاه بدم دورش بلدم نیستم البته

rezaiyan replied:
اکیه، من انجام میدم
بعد با هم کدشو بررسی میکنیم

rezaiyan commented:
عالی👌
فقط همون نکات طراحی مونده
هر جایی کمک خواستی حتما بگو
مرسی

rezaiyan commented:
برای که اصلاحات رو زودتر انجام بدم من مرج می‌کنم
